include/flexflow/substitution.h

```diff
@@ -128,6 +128,17 @@ class GraphCompare {
   }
 };
 
+class GraphCompareWithMemory {
+  float run_time_cost_factor;
```

Comment:
```suggestion
private:
  float run_time_cost_factor;
```

Also, most of the FF code puts the variable declarations at the bottom of the class. I'm all up for discussing changing this, but for this PR let's match the existing style

Reply:
Updated